docs/source/guidelines/languages_frameworks.md

```diff
@@ -79,5 +79,5 @@ User-editable configuration files should use the YAML format. As far as possible
 
 ### Release
 * All Python software should be released on PyPI to enable a simple `pip install`. At a minimum, each project should 
-* have two owners - [Adam (adamltyson)](https://github.com/adamltyson) and the lead developer. Others can be added as owners/maintainers as appropriate.
+* Have two owners - [Adam (adamltyson)](https://github.com/adamltyson) and the lead developer. Others can be added as owners/maintainers as appropriate.
```

Comment:
should continue from the previous line

Reply:
Sphinx will ignore these new lines I think (unless theres whitespace), and I think it improves readability of the source (if the editor doesn't wrap text).

Reply:
sorry I'm referring to the capitalisation

Reply:
Fixed now.